portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* Shows about text when the "About" button is pressed. */
+function showAbout () {
+    const aboutContainer = document.getElementById('aboutSpace');
```

Comment:
I believe JS should have a default indentation of 2 spaces as opposed to 4.  Can you update this file's indentation?